app-i18n/fcitx-bamboo/fcitx-bamboo-1.0.1.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg cmake
+
+MY_PN="fcitx5-bamboo"
+MY_P="${MY_PN}-${PV}"
+S="${WORKDIR}/${MY_PN}-${PV}"
+DESCRIPTION="Typing Vietnamese by Bamboo core engine for Fcitx5"
+HOMEPAGE="https://github.com/fcitx/fcitx5-bamboo"
+
+SRC_URI="https://download.fcitx-im.org/fcitx5/${MY_PN}/${MY_P}.tar.xz"
+
+LICENSE="GPL-2+"
```

Comment:
Incorrect License, should be `LGPL-2.1+` 

refer to https://github.com/fcitx/fcitx5-bamboo/tree/main/LICENSES

Reply:
And, please also add `MIT`  due to the bamboo-core license: https://github.com/BambooEngine/bamboo-core/blob/master/LICENSE

Reply:
Edited. Please re-check to help me ~~!